packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -64,170 +57,175 @@ describe("create-turbo cli", () => {
   });
 
   afterAll(() => {
-    execSync("corepack enable", { stdio: "ignore" });
-
     // clean up after the whole test suite just in case any excptions prevented beforeEach callback from running
     cleanupTestDir();
   });
 
-  beforeEach(() => {
-    // cleanup before each test case in case the previous test timed out.
-    cleanupTestDir();
-  });
-
-  afterEach(() => {
-    // clean up test dir in between test cases since we are using the same directory for each one.
-    cleanupTestDir();
-  });
-
   describe("--no-install", () => {
-    it("default: guides the user through the process", async () => {
-      configurePackageManager(PACKAGE_MANAGERS["npm"]);
-      const cli = spawn("node", [createTurbo, "--no-install"], {});
-
-      const messages = await runInteractiveCLI(cli);
-
-      expect(messages[0]).toEqual(
-        ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
-      );
+    it.concurrent.each(Object.values(PACKAGE_MANAGERS).flat())(
+      `--use-$command: guides the user through the process ($name)`,
+      async (packageManager) => {
+        configurePackageManager(packageManager);
+        let testDir = `my-${packageManager.name}-no-install-turborepo`;
+        const cli = spawn(
+          "node",
+          [createTurbo, "--no-install", `--use-${packageManager.command}`],
+          { cwd }
+        );
 
-      expect(messages[1]).toEqual(
-        `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
-      );
+        const stdout = await runInteractiveCLI(cli, testDir);
 
-      expect(getPromptChoices(messages[2])).toEqual(["npm", "pnpm", "yarn"]);
+        expect(stdout).toContain(
+          ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+        );
```

Comment:
👍🏾 for not caring about order of output. It was a huge pain to find the right line number (especially because it also seemed to change if I had `console.log`s for debugging